src/main/java/com/j256/simplemagic/entries/IanaEntry.java

```diff
@@ -0,0 +1,122 @@
+package com.j256.simplemagic.entries;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * <a href="https://www.iana.org/assignments/media-types/media-types.xhtml">IANA metadata</a> coming from
+ * @author Jean-Christophe Malapert (jcmalapert@gmail.com)
+ */
+public class IanaEntry {
+    
+    private static final String MIME_TYPE_BASE_URL = "https://www.iana.org/assignments/media-types/";
+    private static final String RFC_REFERENCE_BASE_URL = "https://tools.ietf.org/html/";    
+    private static final String MIME_TYPE_REFERENCE_BASE_URL = "https://www.iana.org/assignments/media-types/media-types.xhtm";
+    private static final Pattern PATTERN_REGEX = Pattern.compile("\\[(.+?)\\]");
+    
+    /**
+     * Name of the file type
+     */
+    private final String _name;
+    
+    /**
+     * Mime type
+     */
+    private final String _mimeType;
+    
+    /**
+     * URL describing the mime type
+     */
+    private final String _mimeTypeUrl;
+    
+    /**
+     * Reference describing the mime type such as RFC
+     */
+    private final List<String> _reference;       
+    
+    /**
+     * URL of the reference
+     */
+    private List<String> _referenceURL;       
+    
+    public IanaEntry(final String name, final String mimeType, final String reference) {
+        this._name = name;
+        this._mimeType = mimeType;
+        this._reference = parseReference(reference);
+        this._mimeTypeUrl = MIME_TYPE_BASE_URL+_mimeType;
+        this._referenceURL = buildUrl(_reference);
+    }
+    
+    /**
+     * Parse the references and creates one entry by reference.
+     * @param reference
```

Comment:
Please consider removing worthless comments like these.  They don't add anything to the code and make the programming think that they are real comments.